save-frontend/webpack.config.d/dev-server.js

```diff
@@ -1,54 +1,14 @@
 config.devServer = Object.assign(
```

Comment:
Do we now want to make everyone start a gateway to test locally?

Reply:
removed it. I merged it by accident.